radionice/pocetni/lekcije/003_imenice_i_pokazne_zamjenice/003_imenice_i_pokazne_zamjenice.tex

```diff
@@ -59,32 +59,42 @@
 	\end{tabular}
 	\vspace{10pt}
 	
-	Zamislimo da u ruci držimo jabuku. Zatim nam dođe netko i pita nas što je to.
-	Tada možemo reći \textit{\underline{これ}　は　りんご　です。 Ovo je jabuka}. Ako tu istu jabuku dodamo toj drugoj osobi tada možemo reći:	\textit{ \underline{それ}　は　りんご　です。 To je jabuka.} Ako smo ukrali tu jabuku i ne želimo da nas ulove možemo baciti jabuku u susjedovo dvorište i reći: \textit{\underline{あれ}　は　りんご　です。Ono je jabuka.}
+	Zamislimo da u ruci držimo jabuku. Zatim nam dođe netko i pita nas što je to. Tada možemo reći \textit{\underline{これ}　は　りんご　です。 Ovo je jabuka}.\newline 
```

Comment:
Ovo mi i dalje izgleda ruzno u pdf-u. Mozda da primjere na japanskom stavim u novi red? Iako to trosi ekstra papira.

Reply:
U redu je primjere stavljati u novi red, zapravo uglavnom tako i radimo jer je visina jap. teksta nešto malo iznad latinice pa unakazi vertikalni prored.